packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.styles.ts

```diff
@@ -91,9 +93,51 @@ export const StyledSvgControlsSettingsSolid = styled(SvgControlsSettingsSolid)`
 export const StyledSvgControlsSettingsOutline = styled(SvgControlsSettingsOutline)`
   ${defaultIconColor};
 `
+export const StyledSvgControlsShare = styled(SvgControlsShare)`
+  ${defaultIconColor};
+`
 
 export const TRANSITION_DELAY = '50ms'
 
+export const EmbbeddedTopBar = styled.div<{ isFullScreen: boolean }>`
+  font-size: ${sizes(4)};
+  width: 100%;
+  z-index: ${zIndex.overlay};
+  position: relative;
+  background: linear-gradient(180deg, rgb(11 12 15 / 0.9) 0%, transparent 100%);
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  align-items: center;
+  padding: 0.5em 0.5em 0.5em 1em;
+  gap: 1em;
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity, visibility;
+  transition-delay: ${TRANSITION_DELAY};
+  transition-duration: 200ms;
```

Comment:
Couldn't we use some design token here with similar values?

Reply:
These values are quite old. Some of them were passed to me during calls, they were never documented if I'm not mistaken. If I change that we would need to go through all the animations in the player and make sure that everything works as expected. We'd need to update also easings and delays. I don't want to bother designers about this right now. 
For now, let's just keep the same values as we have in the controls overlay until we get proper documentation for the player, wdyt?